jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -149,35 +149,43 @@ private static void assertLayerSize(int expected, String imageReference)
 
   @Rule public final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-  private final ProgressChecker progressChecker = new ProgressChecker();
+  private ProgressChecker progressChecker = new ProgressChecker();
 
   @Test
   public void testSteps_forBuildToDockerRegistry()
       throws IOException, InterruptedException, ExecutionException, RegistryException,
-          CacheDirectoryCreationException {
+          CacheDirectoryCreationException, InvalidImageReferenceException {
+    System.setProperty("jib.alwaysCacheBaseImage", "true");
+    String imageReference = "localhost:5000/testimage:testtag";
+    Path cacheDirectory = temporaryFolder.newFolder().toPath();
```

Comment:
Do we want to enable caching for the other tests as well?

Reply:
Or make `line 157` and `line 160` part of `setUp()`

Reply:
Caching is always enabled. It's just that this particular test purposely gives a new empty directory to not reuse any existing cache. (The test times consecutive builds to ensure the second build runs faster due to caching from the first run.)

Reply:
Oh gotcha, that makes a lot of sense. Thanks!